README.md

```diff
@@ -4,97 +4,164 @@
 
    * [snowdrop-automation-client](#snowdrop-automation-client)
       * [Table of contents](#table-of-contents)
-      * [Basics](#basics)
-      * [Running](#running)
-         * [Prerequisites](#prerequisites)
-         * [Launch application](#launch-application)
-      * [Install specific nvm version](#install-specific-nvm-version)
-      * [Miscellaneous](#miscellaneous)
-      * [Run in a container](#run-in-a-container)
-         * [buildah/podman](#buildahpodman)
-            * [Building the container](#building-the-container)
-            * [Running the container](#running-the-container)
-            * [k8s](#k8s)
-         * [Docker](#docker)
-            * [Build with docker-compose](#build-with-docker-compose)
-            * [Run with docker](#run-with-docker)
+      * [Introduction](#introduction)
+      * [Available commands](#available-commands)
+      * [Configuration](#configuration)
+      * [Running the client locally](#running-the-client-locally)
```

Comment:
I'd put how to run the client as one of the first things because that's what you usually need to do first (e.g. you can't try the available commands if you don't first run the client).

Reply:
My thinking was that the commands are the ones that are the most important. Especially if we'll get the client running on Hetzner. But I don't mind swapping the order.

Reply:
I guess the current order is fine if we get the client running on Hetzner…